src/pages/ServiceGraph/SummaryPanelEdge.tsx

```diff
@@ -132,13 +141,46 @@ export default class SummaryPanelEdge extends React.Component<SummaryPanelPropTy
         }
         const data: M.Metrics = response['data'];
         const metrics: Map<String, M.MetricGroup> = data.metrics;
-        const reqRates = this.getRates(metrics['request_count_in'], 'RPS', sourceService, sourceVersion);
-        const errRates = this.getRates(metrics['request_error_count_in'], 'Error', sourceService, sourceVersion);
+        const histograms: Map<String, M.Histogram> = data.histograms;
+        const reqRates = this.getDatapoints(metrics['request_count_in'], 'RPS', sourceService, sourceVersion);
+        const errRates = this.getDatapoints(metrics['request_error_count_in'], 'Error', sourceService, sourceVersion);
+        const latAvg = this.getDatapoints(
+          histograms['request_duration_in']['average'],
+          'Average',
+          sourceService,
+          sourceVersion
+        );
+        const latMed = this.getDatapoints(
+          histograms['request_duration_in']['median'],
+          'Median',
+          sourceService,
+          sourceVersion
+        );
+        const lat95 = this.getDatapoints(
+          histograms['request_duration_in']['percentile95'],
+          '95th',
+          sourceService,
+          sourceVersion
+        );
+        const lat99 = this.getDatapoints(
+          histograms['request_duration_in']['percentile99'],
+          '99th',
+          sourceService,
+          sourceVersion
+        );
+
+        // REMOVE dump latency
+        console.log('METRICS: ' + JSON.stringify(metrics));
+        console.log('HISTOS: ' + JSON.stringify(histograms));
```

Comment:
should we remove these debug messages? Seems like it might produce noisy logs, json-ifying metric and histrogram data.

Reply:
yes, that is why I left that "REMOVE" in there, because I'd likely forget...  They are gone now...